qiskit/dagcircuit/dagnode.py

```diff
@@ -343,7 +343,7 @@ def name(self, new_name):
 
     def __repr__(self):
         """Returns a representation of the DAGOpNode"""
-        return f"""DAGOpNode(name : {self.name}, qargs : {self.qargs}, cargs : {self.cargs})"""
+        return f"DAGOpNode(op = {self.op}, qargs = {self.qargs}, cargs = {self.cargs})"
```

Comment:
This looks better with the equals signs, thanks.  One more nitpicky thing: the standard convention is not to have spaces around the operator (e.g. see what [`collections.namedtuple`](https://docs.python.org/3/library/collections.html#collections.namedtuple) does).  It's good to stick to things like that for consistency - everything in `qiskit` with a similar `repr` (`Instruction`, `Waveform`, `Result`) will do that too.